app/components/Contacts/ContactsPanel/ContactsPanel.jsx

```diff
@@ -20,6 +20,7 @@ import CopyToClipboard from '../../CopyToClipboard'
 import LogoWithStrikethrough from '../../LogoWithStrikethrough'
```

Comment:
I think ideally this file would actually not be updated at all in this PR 👀

Reply:
I agree